### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by chandrikadeb7**
Added me!

Issue #313

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+  {
```
Comment (AllanRegush) [inline data.json:2]: Please add yourself to the bottom :nerd_face:

**Review by AllanRegush — COMMENTED** [review]
Thanks for your PR :smile: Looking Good :+1:

File: data.json (hunk 1 above)
Comment (chandrikadeb7) [reply data.json:2]: Okay!

**Comment by chandrikadeb7** [comment]
> Thanks for your PR 😄 Looking Good 👍

Please review!

File: data.json [hunk 2]
```diff
@@ -550,5 +550,10 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/58151963/107116883-97f86480-6876-11eb-8382-fc1c6ce06403.png",
     "issueNumber": 338
-  }
+  },
+  {
+    "name": "Chandrika Deb",
+    "githubUsername": "chandrikadeb7",
+    "imageUrl": "https://user-images.githubusercontent.com/29686102/106919215-f0711a00-672f-11eb-88f6-8ab44f67b911.png",
+    "issueNumber": 313
 ]
```
Comment (AllanRegush) [inline data.json:559]: It looks like there is a missing closing brace.

**Review by AllanRegush — COMMENTED** [review]
Looking Good 👍 Just a minor adjustment 😄

**Comment by chandrikadeb7** [comment]
> Looking Good 👍 Just a minor adjustment 😄

Done! My bad.

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1:

**Comment by AllanRegush** [comment]
It looks like there a conflict. Could  you fix this? If you need any help let me know. :nerd_face:

**Comment by chandrikadeb7** [comment]
> It looks like there a conflict. Could you fix this? If you need any help let me know. 🤓

Conflicts resolved!

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Comment by eddiejaoude** [comment]
Great team work 👍

**Pull request merged**